woocommerce/woocommerce-functions.php

```diff
@@ -236,3 +237,26 @@ function bootscore_wc_capture_hook_output($hookName) {
   ob_end_clean();
   return $hookContent;
 }
+// Correct hooked checkboxes in checkout End
+
+
+// Redirect to my-account if offcanvas login failed
+add_action( 'woocommerce_login_failed', 'bootscore_redirect_on_login_failed' , 10, 0 );
+function bootscore_redirect_on_login_failed() : void {
```

Comment:
Is there a reason for the `: void` added in this function? Have not seen the usage of that at all, to be honest.

Reply:
No, haven't seen it. If it works without - delete it.